src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,77 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * Achievement collection
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class ModAchievements
+{
+
+    // Achievements
+    public static Achievement achGetSupply      = new AchGetSupply("supply", "supply", -2, -2).registerStat();
+    public static Achievement achWandOfbuilding = new AchWandOfBuilding("wandofbuilding", "wandofbuilding", 0, -2)
+            .registerStat();
+
+    // Buildings
+    public static Achievement achBuildingTownhall = new AchBuildingTownhall("townhall", "townhall", -2, 0)
+            .registerStat();
+
+    public static Achievement achBuildingBuilder   = new AchBuildingBuilder("upgrade.builder.first", "upgrade.builder.first", 0, 1)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static achBuildingBuilder" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1444)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make achBuildingBuilder a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)